examples/xlnet/xlnet_classification_main.py

```diff
@@ -55,9 +65,10 @@ def parse_args():
         help="Directory to save model checkpoints")
 
     parser.add_argument(
-        "--pretrained", type=str,
-        default="pretrained/xlnet_cased_L-24_H-1024_A-16/xlnet_model.ckpt",
-        help="Path to the pretrained XLNet model checkpoint")
+        "--pretrained_model_name", type=str,
```

Comment:
Let's use consistent naming styles. Now we're using `argument-names-like-this`, but not `like_this`.